CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Below is a list of Flutter team members who are suggested reviewers
+# for contributions to plugins in this repository.
+#
+# These names are just suggestions. It is fine to have your changes
+# reviewed by someone else.
+
+packages/**/*_web/**                           @ditman
```

Comment:
I hope this works as advertised! I'll keep an eye out on my auto-assigned issues!

Reply:
It seems to be working! Very nice!